components/azure-kubernetes-service/kafka-topics.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: kafka.strimzi.io/v1beta1
+kind: KafkaTopic
+metadata:
+  name: test
+  labels:
+    strimzi.io/cluster: my-cluster
+spec:
+  partitions: 6
+  replicas: 3
+  config:
+    retention.ms: 7200000
+    segment.bytes: 1073741824
+---
+apiVersion: kafka.strimzi.io/v1beta1
+kind: KafkaTopic
+metadata:
+  name: test-one-rep
```

Comment:
These topic names do not match the ones used in the solution. Can you please explain what is the role of this file?

Reply:
This was just being used to create a sample topic inside the cluster and as soon as we finish spinning up the cluster, we have a topic to take a look at! We don't even need to create this topic before, since as soon as we try to submit a message to a topic it will be created automatically. So both of paths work for me, what do you think its the best?
- Changing this topic creation to the name that we are using in the solution
- Not creating this at all

Reply:
I've deleted the file for now as it was not used, but it would be great to restore it into the helm  chart. In practice, actual projects will usually want to enforce governance around topic lifecycle

Reply:
Awesome, makes sense!